OmniCore.Mobile/OmniCore.Mobile.Android/MainActivity.cs

```diff
@@ -53,6 +55,27 @@ protected override void OnCreate(Bundle savedInstanceState)
             IsCreated = true;
         }
 
+        private void ToggleAdapter()
+        {
+            try
+            {
+                if (CrossBleAdapter.Current.CanControlAdapterState())
+                {
+                    // Switch adapter on if off (testing)
+                    if (CrossBleAdapter.Current.Status == AdapterStatus.PoweredOff)
+                    {
+                        CrossBleAdapter.Current.SetAdapterState(true);
+                        Wait(5000);
```

Comment:
why are we waiting here? and what if the user declines the request - further checks? shutdown app? etc.

Reply:
Yes. * Please ignore! *
Actually a bit of test code, looking for a way to "auto-toggle" BT as this seemed to way to regain connectivity.

Left it in as I wanted to leave the code identical to what i tested with.

Btw: I do not see any user request. as wondering if this should be programmed additionally.